ext/fetch/23_request.js

```diff
@@ -455,13 +429,8 @@ class Request {
     if (this[_method]) {
       return this[_method];
     }
-    if (this[_flash]) {
-      this[_method] = this[_flash].methodCb();
-      return this[_method];
-    } else {
-      this[_method] = this[_request].method;
-      return this[_method];
-    }
+    this[_method] = this[_request].method;
+    return this[_method];
```

Comment:
This caching seems iffy. I bet this is one of the reasons for excessive garbage collection on the flamegraphs. I dare to say it's faster to always access `this[_request].method` directly.